Several small improvements

Yatanvesh opened the pull request:
1. Accommodate RN 0.60 and above projects, bump build tool version and targetSdkVersion.
2. Decrease minSdkVersion to 19, and in doing so added a check in  enterPictureInPictureMode to ignore sdk versions 23 and below.
3. Change package identifier to com.reactpiplibrary as com.reactlibrary is already taken by another library, which causes release build to fail.
4. Update readme

Yatanvesh committed:
Upgrade buildToolsVersion->28.0.3, targetSdkVersion->28, downgrade minSdk->19, rename package name to reactpiplibrary to avoid name clash conflicts in release mode

Yatanvesh committed:
Due to minSdk->19  support, added check in enterPictureInPictureMode function to detect sdk>=24 before calling pip native method

Yatanvesh committed:
Bump package version no, change readme to accomodate changing times

shaun-chiang reviewed (approved):
hey @Yatanvesh, thanks so much for helping with this :) LGTM

Yatanvesh commented:
I'm using this in my project, it took me a while to get it working, but when it did, I decided to do it properly and commit.
Although simple, thank you for creating this library

Yatanvesh committed:
Added aspect ratio configure, automatic pip mode switch functionality

Yatanvesh commented:
Hi, added functionality to configure aspect ratio( Keeping android SDK level restriction in mind), and auto PIP switch functionality option when app goes out of focus, updated readme 
(A typo went into prev commit, fixed it in next)

Yatanvesh committed:
minor
